src/core/reference/include/openvino/reference/round.hpp

```diff
@@ -0,0 +1,81 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <cfenv>
+#include <cmath>
+#include <type_traits>
+
+#include "openvino/op/round.hpp"
+
+namespace ov {
+template <class T>
+constexpr bool is_floating_point() {
```

Comment:
It can be common util function

Reply:
Where should be stored in project in (e.g element_type_traits)?

Reply:
It can be a dedicated header in the reference library.

Reply:
OK, Will remember about it and do it next time or if there will any new changes applied into this branch (because CI pass so don't want trigger re-run because of new source file)